client.go

```diff
@@ -302,10 +315,16 @@ func (c *TunaSessionClient) getPubAddrs(includePrice bool) *PubAddrs {
 				addr.InPrice = entryToExitPrice.String()
 				addr.OutPrice = exitToEntryPrice.String()
 			}
+			addrs = append(addrs, addr)
 		}
-		addrs = append(addrs, addr)
 	}
-	return &PubAddrs{Addrs: addrs}
+
+	pubAddrs := &PubAddrs{}
```

Comment:
We need to make sure non empty pubaddrs is returned as long as one tuna exit is ready

Reply:
Yes. modified.